gpkit/model.py

```diff
@@ -176,7 +176,7 @@ def zero_lower_unbounded_variables(self):
                      if bound == "lower"}
             self.substitutions.update(zeros)
 
-    def solve(self, solver=None, verbosity=1, skipfailures=True,
+    def solve(self, solver=None, verbosity=1, skipfailures=False,
```

Comment:
Excellent, that makes sense.

The documentation for the `skipfailures` argument is currently 
```python
        skipfailures : bool (optional)
            If True, when a solve errors during a sweep, skip it.
```
Should this be updated to include interactives instead of just sweeps?